src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
```

Comment:
Done! In the last commit I replaced the use of `Array[Byte]` for `IndexedSeq[Byte]` as the `Key` and `Value` of `DataSource`